### DISCUSSION TITLE
使用[ObservableProperty]需要什么额外的配置吗？我用它并不会自动生成相应的代码

### DISCUSSION THREAD

**Discussion by dianxu2357** [BYJRK/DotNet-Discussions#47] [MVVM]
我的viewmodel已经继承了ObservableObject，并且声明了partial
<img width="1031" alt="observableproperty" src="https://user-images.githubusercontent.com/29402733/223441439-4fac5ef3-bf7f-4573-bbc0-22ce814d1af0.png">

**Comment by BYJRK** [comment]
你是 Framework 项目吗？或者 .NET 版本低于 5.0 的

**Reply by dianxu2357** [reply]
谢谢，我明白了
